packages/desktop/package.json

```
@@ -1,11 +1,11 @@
 {
     "name": "desktop",
-    "productName": "Firefly Shimmer",
-    "version": "2.1.3",
-    "description": "Official wallet application of Shimmer",
+    "productName": "Bloom",
+    "version": "1.0.0",
+    "description": "Simple and secure web3 wallet centered around IOTA and Shimmer",
```

If I may make a suggestion: 

```suggestion
    "description": "Simple and secure web3 wallet for the IOTA and Shimmer ecosystem",
```